src/components/views/settings/ThemeChoicePanel.tsx

```diff
@@ -168,7 +168,7 @@ export default class ThemeChoicePanel extends React.Component<IProps, IState> {
             (findHighContrastTheme(this.state.theme) || isHighContrastTheme(this.state.theme))
         ) {
             return (
-                <div>
+                <div data-testid="checkbox-use-high-contrast">
```

Comment:
Can you try to find this element with `cy.findByText("Use high contrast")` instead of adding a test Id? Maybe even `findByRole()` could work. But I am not sure in this particular place.